google-cloud-bigquerystorage/src/test/java/com/google/cloud/bigquery/storage/v1/StreamWriterTest.java

```diff
@@ -343,6 +344,20 @@ public void testAppendFailedSchemaError() throws Exception {
     writer.close();
   }
 
+  @Test
+  public void testAppendFailRandomException() throws Exception {
+    StreamWriter writer = getTestStreamWriter();
+    // Trigger a non-StatusRuntimeException for append operation
```

Comment:
You can add a comment saying GRPC API shouldn't return anything other than StatusRuntimeException, just being extra careful about unexpected behavior.

Reply:
Done.